### PR TITLE
Wizards are now at least 30 years old. Apprentices are 17 to 29.

### PR DESCRIPTION
**Pull request by ghost**
:cl: McBawbaggings
fix: Wizards are now at least 30 years old. Apprentices will be in-between ages 17 to 29.
/:cl:

Creeperjoe said wizards get set to age 30. They don't, this is clearly an oversight.

### CODE REVIEW

**Comment by kevinz000** [comment]
dude meme lmao

**Comment by Bawhoppen** [comment]
high priority PR

**Comment by doobiedoo23** [comment]
Judging by their power level the Wizard Federation wizards are at least in their 40s

**Comment by Shapsy** [comment]
I thought wizardry came at 40

**Comment by Repukan** [comment]
Comes at 30, liches is 40

File: code/game/gamemodes/wizard/wizard.dm [hunk 1]
```diff
@@ -110,6 +110,10 @@
 		wizard_mob.name = newname
 		if(wizard_mob.mind)
 			wizard_mob.mind.name = newname
+
+		/* Wizards are at least 30 years old */
+		if(wizard_mob.age < 30)
+			wizard_mob.age = 30
```
Comment (Cyberboss) [inline code/game/gamemodes/wizard/wizard.dm]: Make this a define WIZARD_AGE_MIN next to AGE_MIN

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/wizard/wizard.dm (hunk 1 above)
Comment (McBawbaggings) [inline code/game/gamemodes/wizard/wizard.dm]: Aight

**Comment by Bawhoppen** [comment]
this PR actually enrages me the more I think about it

**Comment by ohnopigeons** [comment]
But what if I want to play a teenage magical girl

**Comment by McBawbaggings** [comment]
I actually thought of having different checks for guys and girls.
Guys would have the same change but girls would be like:
```M.age = /* 3000 */ 17```

But I'm actually slightly serious with the PR so I refrained.

**Comment by ChangelingRain** [comment]
apprentices should be at LEAST 30, wizards should have a minimum of like 45 or something

**Comment by McBawbaggings** [comment]
I disagree but others seem to echo the idea, I'll probably change the numbers about. 30 is the meme age and I'd like to preserve it.

**Review by Cyberboss — APPROVED** [review]

**Comment by Bawhoppen** [comment]
yea I don't see why we need them to be old and gross
wizards can be young and cool

**Comment by Jordie0608** [comment]
\>Defines for wizard age

Can players even see this anywhere in game?

**Comment by Bawhoppen** [comment]
no but we know it

**Comment by Nabski89** [comment]
Eternal youth gives different messages based on age, so that's a way to tell player age in game.

**Comment by Bawhoppen** [comment]
you're a fucking monster leo

**Comment by lzimann** [comment]
i'm just following the lore

**Pull request merged**